### PR TITLE
Add a small test for IsVolumeAnEmptyDir

### PR DESCRIPTION
**Pull request by cdrage**
Adds a small test for IsVolumeAnEmptyDir.

Finishes (part) of https://github.com/redhat-developer/odo/issues/828

### CODE REVIEW

File: pkg/occlient/occlient_test.go [hunk 1]
```diff
@@ -3221,3 +3221,49 @@ func TestUpdateDCToSupervisor(t *testing.T) {
...
+func TestIsVolumeAnEmptyDir(t *testing.T) {
+	type args struct {
+		VolumeName string
+		dc         appsv1.DeploymentConfig
+	}
+	tests := []struct {
+		name         string
+		args         args
+		wantEmptyDir bool
+		actions      int
```
Comment (kadel) [inline pkg/occlient/occlient_test.go]: It looks like that `actions` is not used anywhere

File: pkg/occlient/occlient_test.go [hunk 2]
```diff
@@ -3221,3 +3221,49 @@ func TestUpdateDCToSupervisor(t *testing.T) {
...
+		},
+		{
+			name: "Case 2 - Check a non-existent volume",
+			args: args{
+				VolumeName: "foobar",
+				dc:         *fakeDeploymentConfig("foo", "bar"),
+			},
+			wantEmptyDir: false,
+			actions:      1,
+		},
```
Comment (kadel) [inline pkg/occlient/occlient_test.go]: there should probably be one more test case for volume that exists, but it is something else then emptyDir

**Comment by cdrage** [comment]
@kadel 

Updated with the changes you requested (new case added, removes `actions` param)

**Comment by mik-dass** [comment]
@kadel Can we merge this?

**Comment by kadel** [comment]
> @kadel Can we merge this?

sure
I will leave mergin on @cdrage so he can reply to  @anmolbabu's comment

**Comment by cdrage** [comment]
Hey @anmolbabu I had to keep it in the same occlient_test.go file since the supervisorD bootstrap templates are located within the same package.

**Pull request merged**